index.js

```diff
@@ -22,6 +24,27 @@ app.use(require("morgan")("dev"));
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 
+// Swagger definition
+const swaggerDefinition = {
+    info: {
+      title: 'REST API for Authors Haven',
+      version: '1.0.0',
+      description: 'This is the REST API for Authors Haven',
+    },
+    host: 'localhost:3000',
+    basePath: '/',
+  };
+
+// options for the swagger docs
+const options = {
+  // import swaggerDefinitions
+  swaggerDefinition: swaggerDefinition,
+  // path to the API docs
+  apis: ['./docs/swagger/**/*.js'],
```

Comment:
Use a yaml file for the docs descriptions. This becomes `apis: ['./docs/swagger/**/*.yaml']`. So change all .js file in `./docs/swagger/` path to `.yaml`. So we doc description like
```
/**
 * @swagger
 * definitions:
 *   signUp:
 *     properties:
 *       username:
 *         type: string
```
becomes
```
definitions:
 signUp:
   properties:
     username:
       type: string
```